omaha/hammer-brave.bat

```diff
@@ -84,7 +84,7 @@ set OMAHA_PLATFORM_SDK_DIR=%WindowsSdkDir%\
 set OMAHA_WINDOWS_SDK_10_0_VERSION=%WindowsSDKVersion:~0,-1%
 
 :: Directory which includes the sign.exe tool for Authenticode signing.
-set OMAHA_SIGNTOOL_SDK_DIR=%WindowsSdkDir%\bin\10.0.15063.0\x86
```

Comment:
Has `signtool.exe` moved in more recent versions of the SDK? On my laptop, it's in `bin/10.0.17763.0/x86/signtool.exe`. I don't understand how it could be in a top-level directory like this change is implying, actually. Because then what would happen if you had multiple SDKs installed?

Reply:
This change was back to original `hammer.bat` setting.
I'm also curious why I have it in `10/bin/x86`. You don't have it?

Reply:
actually this change is not directly related with issue what this PR want to fix.
Just want to remove specifying specific sdk version.